tools/@aws-cdk/pkglint/lib/rules.ts

```diff
@@ -1708,6 +1708,12 @@ export class NoExperimentalDependents extends ValidationRule {
         return;
       }
 
+      // eslint-disable-next-line @typescript-eslint/no-require-imports
+      const maturity = require(`${dep}/package.json`).maturity;
+      if (maturity === 'cfn-only') {
+        return;
+      }
+
       // eslint-disable-next-line @typescript-eslint/no-require-imports
       const stability = require(`${dep}/package.json`).stability;
```

Comment:
We should ignore `stability`.

* `maturity` == how we feel about the reliability of the package.
* `stability` == the default stability of API elements in this package (i.o.w, setting this is just syntactic sugar over putting `@stable`, `@experimental` etc in every docstring)